### PR TITLE
Register a new HostedCE for KSU Beocat cluster

### PR DESCRIPTION
**Pull request by Mansalu**
IDs needed for group and for the CE (not sure how those are generated)

Let me know if there are any issues.

Thanks!!

### CODE REVIEW

File: topology/Kansas State University/Beocat/BEOCAT-SLATE.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+GroupDescription: Kansas State University Beocat HPC Group
+GroupID: <NEED ID>
```
Comment (Mansalu) [inline topology/Kansas State University/Beocat/BEOCAT-SLATE.yaml:2]: Needs group ID

File: topology/Kansas State University/Beocat/BEOCAT-SLATE.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        Primary:
+          ID:  0a077233ccf64c5f4a0888b0de224093f5129e9a
+          Name: Kyle Hutson
+      Site Contact:
+        Primary:
+          ID:  0a077233ccf64c5f4a0888b0de224093f5129e9a
+          Name: Kyle Hutson
+    Description: Hosted CE for the Beocat cluster
+    FQDN: slate-osg.beocat.ksu.edu
+    ID: <NEED ID>
```
Comment (Mansalu) [inline topology/Kansas State University/Beocat/BEOCAT-SLATE.yaml:22]: Needs resource ID

File: topology/Kansas State University/Beocat/BEOCAT-SLATE.yaml (hunk 1 above)
Comment (matyasselmeci) [inline topology/Kansas State University/Beocat/BEOCAT-SLATE.yaml:2]:
```suggestion
GroupID: 1107
```

File: topology/Kansas State University/Beocat/BEOCAT-SLATE.yaml (hunk 2 above)
Comment (matyasselmeci) [inline topology/Kansas State University/Beocat/BEOCAT-SLATE.yaml:22]:
```suggestion
    ID: 1124
```

File: topology/Kansas State University/Beocat/BEOCAT-SLATE.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+GroupDescription: Kansas State University Beocat HPC Group
```
Comment (matyasselmeci) [inline topology/Kansas State University/Beocat/BEOCAT-SLATE.yaml:1]:
```suggestion
SupportCenter: Self Supported
GroupDescription: Kansas State University Beocat HPC Group
```

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**